### ISSUE TITLE
Popup Flicker on mobile

### ISSUE THREAD

**Issue by sgtscience** [elementor/elementor#13180]
## Prerequisites
- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x] The issue still exists against the latest stable version of Elementor.

There was another issue reported by user codeorlov "Bugged popup. #11555". This issue was closed and marked as duplicate back in June. It was not resolved (I am having the same issue) and was not a duplicate of the other issue.

## Description
I made an off canvas menu using popups and slide out right exit animation. More then half the time when closing the menu on my android phone using Chrome and Samsung internet, after the animation ends the menu flickers back to the open position for a split second. This also happens occasionally on desktop using Chrome. I do not have an apple phone to test on. I have tried this with multiple animations like fade out right and fade out and it persists. It seems worse when the close button it pressed very quickly after the menu opens, but this still happens when setting the close button to appear after 2 seconds. I have tried with all other plugins deactivated and the theme twenty twenty installed and it persists.

I have built a site with a vanilla Wordpress install, hello Elementor theme, and Elementor to show the issue. Use the hamburger menu on mobile, you will see the flicker. There is a button on the bottom of the page that activates another popup that also produces the flicker on close. It happens infrequently on this test site, however it occurs at least 1 in 2 times on the site I am working on. I do not want to share that link publicly.

## Steps to reproduce

Install Wordpress.
Install Elementor plugin
Install Elementor Pro plugin
Create a popup with entrance and exit animations
Create a button and under link setting choose dynamic tag > actions > popup and choose the popup
publish the popup and the page with the button
press the button to activate the popup then close it.

## Isolating the problem

- [x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x ] This bug happens with a default WordPress theme active.
- [x ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.28
	PHP Version: 7.4.11
	PHP Max Input Vars: 1000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5.3
	Site URL: http://elementor.sgtscience.com
	Home URL: http://elementor.sgtscience.com
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 256M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-CA
	Timezone: America/Toronto
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.3.0
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_CA
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.66 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.0.14
		Author: Elementor.com

	Elementor Pro
		Version: 3.0.5
		Author: Elementor.com


== Elements Usage ==
	
	header : 1
		icon : 1
		text-editor : 1
	popup : 2
		image : 1
		social-icons : 1
		text-editor : 2
		theme-site-title : 1
	wp-page : 1
		button : 1
		text-editor : 1


== Log ==
	
PHP: showing 2 of 2PHP: 2020-10-27 23:46:07 [notice X 1][/home/dh_wpzxcm/elementor.sgtscience.com/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-27 23:51:00 [notice X 1][/home/dh_wpzxcm/elementor.sgtscience.com/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::162] Undefined index: editor_post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 1 of 12020-11-26 16:55:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.12',
  'to' => '3.0.14',
)]
```
</details>

**Comment by ofmarconi** [comment]
Hello I already use elementor on several sites and I always had this problem with the popup output animation: https://youtu.be/zjqSMh9bygQ

It doesn't always happen, I had to try it 10 times to be able to record this one.

It looks like the animation ends before the popup actually disappears ...

**Comment by riadcitaku** [comment]
@sgtscience I think I've found the solution, you can check it here: https://github.com/elementor/elementor/issues/14459

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.3.0

Feel free to update

Thanks!

**Comment by Jlazar1996** [comment]
Hi,

I am having a similar issue with entrance animations on pop-ups. Not something I've encountered before and only seems to be occurring with the 'slide in' animation. Fade in left instead of slide in left for example it works ok. 

Anyone else finding this?
